docs/source/alignment.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>do you explain that init 90 usually means initialized Dec 31st 1990=Jan1st1991 or November1st1990 in the annual context. and then what this would mean in the monthly context? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>